### ISSUE TITLE
objectness loss is high and the loss can't go down

### ISSUE THREAD

**Issue by G-Apple1** [ultralytics/yolov5#4503]
## ❔Question

the confidence of the detect object is very low, about 0.3

How can i improve the confidence of the detect object?

**Comment by iceisfun** [comment]
Usually this is a result of incorrect or insufficient training data.

Try and add more samples and validate that your existing samples have a correct label.

Also see below, adding background images that do not include labeled objects can help improve training.

If you are still having trouble with sufficient data how many epochs and what is your batch size when training?

https://docs.ultralytics.com/guides/model-training-tips/

**Comment by G-Apple1** [comment]
> Usually this is a result of incorrect or insufficient training data.
> 
> Try and add more samples and validate that your existing samples have a correct label.
> 
> Also see below, adding background images that do not include labeled objects can help improve training.
> 
> If you are still having trouble with sufficient data how many epochs and what is your batch size when training?
> 
> https://docs.ultralytics.com/guides/model-training-tips/

i follow the guide, i think i should add some background images to the train set and train more epochs.

There are not pure background images in my datasets. i want to know how to get the background images. 

Is it to erase the objects on the images？

**Comment by iceisfun** [comment]
Probably not, you should have background images that naturally occur and do not contain any classes in your data set.

Also training for more epochs might not be a good idea, sometimes even training for less can be a good place to start esp when you have little data.

**Comment by pderrenger** [comment]
You're right; using naturally occurring background images without labeled objects is ideal. Consider sourcing images from similar environments. Adjusting the number of training epochs based on your dataset size can also help.
